src/models/transformer.h

```diff
@@ -580,6 +582,12 @@ class EncoderTransformer : public Transformer<EncoderBase> {
       checkpoint(layer); // sets a manually specified checkpoint if gradient checkpointing is enabled, does nothing otherwise.
     }
 
+    // This allows to run a final layernorm operation after going through the transformer layer stack.
+    // By default the operations are empty, but with prenorm (--transformer-preprocess n --transformer-postprocess da) 
+    // we should normalize here. Can also be used to add a skip connection from the very bottom if requested.
```

Comment:
I found that "we should normalize here" leaves the comment a little unclear. Is it a recommendation or todo, and who is "we" referring to, a user or developer? Consider making it clearer.

Reply:
Hm, I don't read it like that. It is actually a recommendation and not a requirement. All of this is quite fuzzy.